docs/tutorial_tx_service_initiate_sign.md

```diff
@@ -194,6 +194,8 @@ print(signature.signature.hex())
 Signature is
 `0xc0df6a1b659d56d3d23f66cbd1c483467ea68a428fea7bbbe0a527d43d8681f616af33344035f36c08218718480374dada0fe6cdb266d0182a4225d0e9c227181b`.
 
+More than one signature (for different owners) can be sent. They should be appended and result should be an hex string `0x` prefixed. [More information on Gnosis Safe contract signatures](contracts_signatures).
```

Comment:
> They should be appended and result should be an hex string `0x` prefixed.

Better spelling would be: "They should be appended and **the** result should be **a** hex string `0x` prefixed."

Also, what about a simple example. Something like:

For 2 signatures `0x11...11` and `0x22...22` the combined signature should be `0x11...1122...22`.

Reply:
Ok!

> Also, what about a simple example. Something like:

That's because of what @rmeissner said about being different depending on the type of signature. For EOAs it's just concatenation, but for contract signatures is not

Reply:
ah in the example on the bottom https://docs.gnosis.io/safe/docs/contracts_signatures/#examples ok!